prog/expr_test.go

```diff
@@ -160,14 +160,14 @@ func TestConditionalMinimize(t *testing.T) {
 			pred: func(p *Prog, _ int) bool {
 				return bytes.Contains(p.Serialize(), []byte("0xaa"))
 			},
-			output: `test$conditional_struct_minimize(&(0x7f0000000040)={0x1, @value=0xaa})`,
+			output: `test$conditional_struct_minimize(&(0x7f0000000040)={0x1, @value=0xaa, 0x1, @value=0xbb})`,
```

Comment:
These test cases make little sense now. Maybe run these tests in the MinimizeCrashSnapshot mode? Or delete these two?

Reply:
Done. PTAL.